docs/getting-started-with-django.asciidoc

```diff
@@ -0,0 +1,299 @@
+[[get-started-with-django]]
+== Get started with Django
+
+Getting Elastic APM set up for your Django project is easy, and there are various ways you can tweak it to fit to your needs.
+
+[[installation]]
+=== Installation
+
+Install the Elastic APM module using pip:
+
+[source,bash]
+----
+$ pip install elastic-apm
+----
+
+or add it to your project's `requirements.txt` file.
+
+NOTE: If you use Django with uwsgi, make sure to
+http://uwsgi-docs.readthedocs.org/en/latest/Options.html#enable-threads[enable
+threads].
+
+[[setup]]
+=== Setup
+
+Set up the Elastic APM module in Django with these two steps:
+
+1.  Add `elasticapm.contrib.django` to `INSTALLED_APPS` in your settings:
+
+[source,python]
+----
+INSTALLED_APPS = (
+   # ...
+   'elasticapm.contrib.django',
+)
+----
+
+1.  Choose an app name, and set the secret token if needed.
```

Comment:
In the future we should link there to what is allowed for app names and what not, but first we need a doc for it.